src/test/java/ch/vorburger/exec/ManagedProcessTest.java

```diff
@@ -223,6 +232,7 @@ public void testMustTerminateExec() throws Exception {
         assertThat(p.isAlive(), is(true));
         p.waitForExitMaxMsOrDestroy(200);
         assertThat(p.isAlive(), is(false));
+        assertThat(p.wasDestroyed(), is(true));
```

Comment:
This is good, but it would be better if you could also add an appropriate assertion everywhere else where isAlive() is already asserted - that would make it really clear how the two differ.

Reply:
Good call. I will amend these.